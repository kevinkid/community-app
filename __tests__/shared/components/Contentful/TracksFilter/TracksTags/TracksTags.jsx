import React from 'react';
import Renderer from 'react-test-renderer/shallow';
import { TracksTagsInner } from 'components/Contentful/TracksFilter/TracksTags';

test('Matches shallow shapshot', () => {
  const renderer = new Renderer();
  const MOCK_PROPS = {
    theme: {},
  };
  renderer.render(<TracksTagsInner {...MOCK_PROPS} />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
