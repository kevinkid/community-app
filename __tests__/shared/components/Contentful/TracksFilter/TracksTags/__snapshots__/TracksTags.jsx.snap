// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches shallow shapshot 1`] = `
<div>
  <span>
    Tags
  </span>
  <div>
    <input
      id="track-tags-input"
      onChange={[Function]}
      onKeyDown={[Function]}
      placeholder="Add tags to filter..."
      type="text"
      value=""
    />
  </div>
</div>
`;
